### DISCUSSION TITLE
Is anyone else getting errors when editing the original interaction?

### DISCUSSION THREAD

**Discussion by Kronicaler** [serenity-rs/serenity#2063] [General]
I've noticed errors when users call a command that's deferred and then calls edit_original_interaction_response()

This just started randomly happening so I'm wondering if it's maybe something wrong with discord.

**Comment by GnomedDev** [comment]
Discord has just rolled over to an extra ID digit, and a lot of bots across multiple libraries have been reporting errors. My guess is that their internal interaction code didn't account for the new ID length

**Reply by Kronicaler** [reply]
Ah thank you. I'm guessing this is fixed in serenity V11? I'm stuck on V10 sadly cause of songbird.

**Reply by GnomedDev** [reply]
it isn't, this is a discord problem not a serenity problem
